packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -507,6 +507,19 @@ mixin SchedulerBinding on BindingBase {
     } // runs next task when there's time
   }
 
+  /// Returns the number of queued tasks in debug mode.
+  ///
+  /// This method is only used for unit tests, and returns 0 in release mode.
+  @visibleForTesting
+  int taskQueueLength() {
```

Comment:
Per convention, the name should begin with the word "debug" if it is a debug-only method.

Reply:
I am not super excited about exposing this in the public API (even when it's `visibleForTesting`). Why do we need this?

Reply:
During my development the scheduler binding tests kept stuck into dead lock, which I later found out is that, due to some bugs in my PR, the task queue is not emptied in some test cases. The task queue should always be empty between tests. I want to verify this at the end of each test to expose this problem early if we ever encounter it again.

Some other alternatives are:
* bool taskQueueEmpty()
* void assertTaskQueueEmpty()   // This is not ideal because we're supposed to use "expect" not assert